packages/gatsby-theme-newrelic/README.md

```diff
@@ -236,6 +237,23 @@ documentation.](https://www.gatsbyjs.org/packages/gatsby-plugin-robots-txt/)
 Optional configuration for related resources used in the right rail. Currently
 only `Mdx` nodes are supported.
 
+The related resources component is controlled by specific Frontmatter slugs that
+are defined on a page. By setting the Frontmatter for `resources` and `tags` you can control what is populated in this component.
```

Comment:
Tags are actually left to the individual site on whether they are used as part of the related resource query to Swiftype or not. The only thing the theme really does out of the box is look for a `resources` field in front matter to pull data from.

Reply:
i'll correct this.